browser-interface/packages/lib/decentraland/profiles/transformations/profileToServerFormat.ts

```diff
@@ -23,6 +23,10 @@ export function ensureAvatarCompatibilityFormat(profile: Readonly<Avatar | OldAv
   avatarInfo.emotes = avatar?.emotes
   avatarInfo.snapshots = avatar?.snapshots
 
+  if (profile.links) {
```

Comment:
this is not related to the PX but the avatar schema validation was throwing errors because the link was `'https%3A%2F%2Fboedo.casla'` instead of `https://boedo.casla`.